adapter/internal/oasparser/model/mgw_swagger.go

```diff
@@ -48,7 +48,7 @@ type MgwSwagger struct {
 	productionEndpoints *EndpointCluster
 	sandboxEndpoints    *EndpointCluster
 	xWso2Endpoints      map[string]*EndpointCluster
-	resources           []Resource
+	resources           []*Resource
```

Comment:
Why are we switching to a pointer?

Reply:
because we later sanitizes populated operations and resources in `SanitizeAPISecurity` line 327.